pages/osx/cot.md

```diff
@@ -0,0 +1,24 @@
+# cot
+
+> The Plain-Text Editor for macOS.
+> More information: <https://coteditor.com/>.
+
+- Start CotEditor:
+
+`cot`
+
+- Open a new blank document:
+
+`cot --new`
+
+- Open specific files:
+
+`cot {{path/to/file1 path/to/file2 ...}}`
+
+- Open and wait until done editing a specific file:
```

Comment:
```suggestion
- Open and wait until finishing editing a specific file:
```

Reply:
Here's my reply from [the other PR](https://github.com/tldr-pages/tldr/pull/8206) for reference:
> This exact line was taken directly from [the `webstorm` tldr-page](https://github.com/tldr-pages/tldr/blob/da55f621eb9c77d579b4353d78cc4c9bec39c4f3/pages/common/webstorm.md), but I agree that "finishing" is better than "done".
> 
> I think "finished" might be a better form of the word than "finishing" to use here, though.